core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockGoogleStorageService.scala

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.dsde.rawls.mock
+
+import cats.data.NonEmptyList
+import com.google.auth.Credentials
+import com.google.cloud.{Identity, Policy}
+import com.google.cloud.storage.{Acl, Blob, BlobId, Bucket, BucketInfo, Storage}
+import fs2.Pipe
+import org.broadinstitute.dsde.workbench.RetryConfig
+import org.broadinstitute.dsde.workbench.google2.{GcsBlobName, GetMetadataResponse, GoogleStorageService, RemoveObjectResult, StorageRole}
+import org.broadinstitute.dsde.workbench.model.TraceId
+import org.broadinstitute.dsde.workbench.model.google.{GcsBucketName, GcsObjectName, GoogleProject}
+
+import java.nio.file.Path
+import scala.language.higherKinds
+
+abstract class MockGoogleStorageService[F[_]] extends GoogleStorageService[F] {
```

Comment:
TOL: We should probably fill these in at some point with what they actually return. Although I am happy to wait until they need to be used for tests. Is that your intention here?

Reply:
I tried but I can't get it to compile!

Reply:
I don't think it's worth it as it's defined by the interface.

Reply:
Why not explicitly throw in that case? My concern is that `???` means to me unimplemented, not intentional.

Reply:
??? is unimplemented:
https://github.com/scala/scala/blob/4b2151cbe4c857d49556b70c3fa0f5e236fd7754/src/library/scala/Predef.scala#L344

This is exactly what I want - if I want to use a method in a mock, you have to implement it.

Reply:
Okay! Sounds like we are actually saying the same thing differently! This works for me!